resources/js/Shared/Layout/AdminNavigation.vue

```diff
@@ -1,66 +1,85 @@
 <script setup>
-import { Link } from '@inertiajs/vue3'
-import { ChartBarIcon, ClipboardIcon, FlagIcon, MapPinIcon } from '@heroicons/vue/24/solid'
-import { Bars3Icon } from '@heroicons/vue/24/outline'
+import { Dialog, DialogPanel } from '@headlessui/vue'
+import { InertiaLink } from '@inertiajs/vue3'
```

Comment:
```suggestion
```
We don`t need this import now.

Also please change `<Link/>` in other views to `<InertiaLink/>` and remove **Link** import,